### DISCUSSION TITLE
Add COBOL to list of languages

### DISCUSSION THREAD

**Discussion by Rune-Christensen** [Genuitec/Copilot4Eclipse-community#52] [Help]
Hi,
Is it possible to add COBOL to the list of languages in the official release?
I have followed the guide on how to add file extensions to an existing language, to add COBOL to my own installation, and it works like a charm.
I have attached my languageId.json file for reference.
Cheers
[languageIds.json](https://github.com/user-attachments/files/17770615/languageIds.json)

Rune

**Comment by wayneparrott** [comment]
@Rune-Christensen Thank you for this feedback. I've shared your request with our team and we are now tracking it in our issues/enhancements system.

Wayne
Copilot4Eclipse Team

**Comment by wayneparrott** [comment]
The Copilot4Eclipse Jan '25 update will include this improvement. Best wishes!

W
